packages/terra-core-docs/src/terra-dev-site/doc/action-header/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,251 @@
+import { Badge } from 'terra-image/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import ActionHeaderExample from "./example/BackActionHeader"
+import BackActionHeader from "./example/BackPreviousNextCloseActionHeader"
+import MaximizeActionHeader from "./example/MaximizeCloseActionHeader"
+import ActionHeadercontainer from "./example/BackActionHeaderContent"
+
+<Badge />
+
+# Accessibility Guide for Terra Action Header
+
+
+### Why is this important?
```

Comment:
Since the above heading is an `h1` this heading should be an `h2` and not `h3`. Remove one `#`.